testing/benchmark/variables.tf

```diff
@@ -83,8 +83,8 @@ variable "apm_shards" {
 ## Worker configuraiton
 
 variable "worker_region" {
-  default     = "us-west-2"
-  description = "Optional ESS region where the deployment will be created. Defaults to us-west-2 (AWS)"
+  default     = "eu-west-3"
```

Comment:
Paris is the closest AWS region from GCP Belgium. The network latency should be negligible